### PR TITLE
Add expect.js@0.3.1 w/ git auto-update & update package.json

### PR DESCRIPTION
**Pull request by him2him2**
cc #11931

Pull request for issue:  #11931
Related issue(s): #11845 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Automattic/expect.js
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/expect.js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 200
   - Count of stars: 1780
   - Count of forks: 189
 * NPM download stats (optional):
   - Downloads in the last day: 10,159
   - Downloads in the last week: 78,585
   - Downloads in the last month: 320,106

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer please help review this PR, thanks.

File: ajax/libs/expect.js/package.json [hunk 1]
```diff
@@ -11,5 +11,14 @@
...
     "type": "git",
     "url": "git://github.com/Automattic/expect.js.git"
-  }
+  },
+  "npmName": "expect.js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "+(expect|index)*.+(js)"
```
Comment (maruilian11) [inline ajax/libs/expect.js/package.json]: Please use `"+(expect|index)*.+(js|css|map)"`. Thanks.

**Comment by maruilian11** [comment]
@him2him2 Seems like the npm package link is not official, could you remove it from init comment?

File: ajax/libs/expect.js/package.json [hunk 2]
```diff
@@ -1,9 +1,26 @@
 {
   "name": "expect.js",
-  "filename": "expect.min.js",
-  "version": "0.2.0",
+  "version": "0.3.1",
+  "license": "MIT",
   "description": "BDD style assertions for node and the browser.",
+  "filename": "index.js",
```
Comment (maruilian11) [inline ajax/libs/expect.js/package.json]: `filename` need to be pointed to a minified file. Please also add the minified file of `index.js`. You can use https://github.com/PeterDaveHello/web-minify-helper, or something famous js compress lib.

File: ajax/libs/expect.js/package.json [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Automattic/expect.js.git"
+  },
   "keywords": [
     "expect.js"
```
Comment (maruilian11) [inline ajax/libs/expect.js/package.json]: Could you please also enrich the `keywords`? Like `BDD-style`, `should.js`. Thanks.

File: ajax/libs/expect.js/package.json [hunk 4]
```diff
@@ -1,9 +1,26 @@
...
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Automattic/expect.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*.js"
```
Comment (maruilian11) [inline ajax/libs/expect.js/package.json]: We usually don't use `**/*` in a non-dist path. Please turn to use `index*.+(js|css|map)`. Thanks.

**Comment by maruilian11** [comment]
`Update expect.js v0.3.1 add autoupdate`
 ---> 
```
Add expect.js@0.3.1 w/ git auto-update

close #11931
```
would  be better. Thanks.

**Comment by sufuf3** [comment]
@maruilian11 Please help review this PR, thanks.
I help modified the auto-update part, and commit message.
If the commit message is not OK, please help improve it.
Many Thanks.

**Comment by sufuf3** [comment]
@extend1994 I  mentioned the minified tool in commit message.

But I found this PR is duplicated with [#11847](https://github.com/cdnjs/cdnjs/pull/11847/files#diff-557a23fca23fa307e975d3b6f16d4dff) which only add repository and license info.
And this PR is late for #11847, So what should we do for this PR?
I think this PR is almost done.
But some part is duplicated with other PR.

@PeterDaveHello Could you tell us how can we do for this PR?
Should we don't touch the repository and license info and wait for #11847 add?

**Comment by extend1994** [comment]
Maybe the sooner merged can do that part. The latter one just remove it?

**Comment by sashberd** [comment]
@sufuf3 I suggest to finish with #11847  and the change this PR to changes that #11847 will insert

**Comment by sashberd** [comment]
@PeterDaveHello please help to review/merge #11847 in order to continue this PR

**Comment by PeterDaveHello** [comment]
@sashberd #11847 merged, let's solve the conflicts here?

**Comment by sashberd** [comment]
@PeterDaveHello All conflict were resolved. FYR and merge

File: ajax/libs/expect.js/package.json [hunk 5]
```diff
@@ -1,14 +1,29 @@
...
+  "license": "MIT",
+  "author": "Guillermo Rauch <guillermo@learnboost.com> (https://zeit.co)",
   "description": "BDD style assertions for node and the browser.",
-  "keywords": [
-    "expect.js"
-  ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/Automattic/expect.js"
+    "url": "git://github.com/Automattic/expect.js.git"
```
Comment (PeterDaveHello) [inline ajax/libs/expect.js/package.json]: We don't need to touch this line.

**Comment by sashberd** [comment]
@PeterDaveHello Done

File: ajax/libs/expect.js/package.json [hunk 6]
```diff
@@ -1,14 +1,29 @@
...
   "description": "BDD style assertions for node and the browser.",
-  "keywords": [
-    "expect.js"
-  ],
+  "author": "Guillermo Rauch <guillermo@learnboost.com> (https://zeit.co)",
   "repository": {
     "type": "git",
     "url": "https://github.com/Automattic/expect.js"
   },
+  "keywords": [
```
Comment (PeterDaveHello) [inline ajax/libs/expect.js/package.json]: Should keep this at line 6, thanks

**Comment by sashberd** [comment]
@PeterDaveHello Done!

**Comment by PeterDaveHello** [comment]
@sashberd THANKS

**Pull request merged**